### ISSUE TITLE
IE bug: opening text category throws js error

### ISSUE THREAD

**Issue by picklesrus** [google/blockly#967]
When trying to open the text category, the following error is thrown:
unable to get property 'textContent' of undefined or null reference on this line:
https://github.com/google/blockly/blob/develop/core/field.js#L332

Before https://github.com/google/blockly/commit/01ab0b4ab6bbae495c09d6ed6f553f9eb9e81b9f, we used to check if textElement was null, but we don't anymore.  The playground works fine though and it includes that change so something else must have changed in between.

**Comment by picklesrus** [comment]
I've narrowed this down a bit.  The blocks that seem to cause trouble are the text blocks that have quotes in them (e.g. the first one in the playground text category). The field that blows up is a FieldImage.

The reason this isn't a problem on chrome or firefox is because of this user agent that guards the check to force a re-render: https://github.com/google/blockly/blob/develop/core/block_render_svg.js#L323. This was added in
https://github.com/google/blockly/commit/d55d9cbd9ff308ff5e361cf77acabb61a4bf4695  I'm guessing textContent just isn't when updateWidth is called that early and we need to wrap it in a if (textContent_) check.

https://github.com/google/blockly/commit/6b692a0802c6a719e547d94420f36c3e847e37c3 also made some changes to the quote blocks and FieldImage, but so far I don't believe is the problem.

Now that I found a way to reproduce on a browser I can run on my dev computer, I should be able to make much quicker progress Monday.
